chapter02/rwd/ex5-2.pl

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/perl
+
+print "Give me a string: ";
+my $s = <>;
+
+print "Give me a number: ";
+my $n = <>;
+
+print "The product is: " . $s x $n ."\n";
```

Comment:
Yeah, the newline is still there so throws them out on separate lines, but the first line has the  "The product is:" printed as well.

I'm just being overly picky, don't mind me.